### PR TITLE
feat(addon-editor): support drop/paste event for resizable

### LINKED ISSUE #3600: Unable to paste an image to resizable image mode / UIKIT-4822

### PR DESCRIPTION
**Pull request by splincode**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring
- [ ] Code style update
- [ ] Build or CI related changes
- [ ] Documentation content changes

## What is the current behavior?

Closes #3600

## What is the new behavior?

https://user-images.githubusercontent.com/12021443/217922035-962be349-1513-4895-acab-00161d21e854.mov

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: projects/demo/src/modules/components/editor/resizable-images/examples/1/index.ts [hunk 1]
```diff
@@ -25,6 +25,12 @@ import {switchMap, takeUntil} from 'rxjs/operators';
                 import('@taiga-ui/addon-editor/extensions/starter-kit').then(
                     ({StarterKit}) => StarterKit,
                 ),
+                import('@tiptap/extension-focus').then(({FocusClasses}) =>
```
Comment (waterplea) [inline projects/demo/src/modules/components/editor/resizable-images/examples/1/index.ts:28]: Why do we need it? CSS :focus-within is not enough?

Comment (splincode) [reply projects/demo/src/modules/components/editor/resizable-images/examples/1/index.ts:28]: I try, it doesn't work :(

Comment (splincode) [reply projects/demo/src/modules/components/editor/resizable-images/examples/1/index.ts:28]: I think it's doesn't work in `contenteditable`

**Review by nsbarsukov — APPROVED** [review]

**Review by vladimirpotekhin — APPROVED** [review]

File: projects/addon-editor/extensions/image-editor/image-editor.extension.ts [hunk 2]
```diff
@@ -78,5 +81,57 @@ export const createImageEditorExtension = (injector: Injector): Node => {
...
+        },
     });
 };
+
+function pasteImage(view: EditorView, event: ClipboardEvent | DragEvent): void {
+    const dataTransfer =
+        event instanceof DragEvent ? event.dataTransfer : event.clipboardData;
+    const hasFiles = dataTransfer?.files?.length ?? 0;
+
+    if (!hasFiles) {
```
Comment (waterplea) [inline projects/addon-editor/extensions/image-editor/image-editor.extension.ts:105]: I think this check is unnecessary, we test for `images.length` later. It only saves us from unnecessary `Array.from` and `filter` which is negligible.

**Review by waterplea — APPROVED** [review]

File: projects/addon-editor/extensions/image-editor/image-editor.extension.ts (hunk 2 above)
Comment (splincode) [reply projects/addon-editor/extensions/image-editor/image-editor.extension.ts]: fixed

**Pull request merged**